src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -80,6 +80,18 @@ bool TwitchMessageBuilder::isIgnored() const
         {
             if (sourceUserID == user.id)
             {
+                switch (getSettings()->showIgnoredUsersMessages)
```

Comment:
making the ShowIgnoredUsersMessages thing an enum class instead would require a static cast here ^